db/migrate/20200303224134_create_extension_tablefunc.rb

```diff
@@ -0,0 +1,5 @@
+class CreateExtensionTablefunc < ActiveRecord::Migration[6.0]
+  def up
+    execute "CREATE EXTENSION tablefunc;"
```

Comment:
The Rails way of adding this extension is `enable_extension "tablefunc"` as it's a little bit smarter.

Reply:
SEE: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/PostgreSQLAdapter.html#method-i-enable_extension 

Then view the source.

Reply:
I'll make the change